cmd/flags.go

```diff
@@ -231,12 +232,16 @@ func clientParameterFlags(v *viper.Viper, cmd *cobra.Command) *cobra.Command {
 		"allow governance to update the client if expiry occurs")
 	cmd.Flags().BoolP(flagUpdateAfterMisbehaviour, "m", true,
 		"allow governance to update the client if misbehaviour freezing occurs")
+	cmd.Flags().Float64(flagClientTrustingPeriod, 0, "custom light client trusting period in hours (default: 85% of unbonding time)")
```

Comment:
So theres a flag type for durations and it will automatically parse a duration in the same format as `time.ParseDuration` -> `cmd.Flags().DurationP()`. This should remove the use of `float64()` conversions in the codebase.

Reply:
Oh nice! 
That float was overcomplicating things.